File: cmstestsuite/code/correct-stdio-inner-class.java
import java.util.Scanner;

public class batchstdio {

    private static class Inner {
        public String solve(int n) {
            return "correct " + n;
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        Inner inner = new Inner();
        System.out.println(inner.solve(n));
    }
}

File: cmstestsuite/code/correct-stdio-inner-class.cs
using System;

public static class Batchstdio
{
    public static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine());
        Console.WriteLine(Inner.Solve(n));
    }

    private static class Inner
    {
        public static string Solve(int n)
        {
            return "correct " + n;
        }
    }
}
